src/assets/userData.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "id": 101,
+    "username": "upaharika"
+  },
+  {
+    "id": 102,
+    "username": "Pratik33"
+  },
+  {
+    "id": 103,
+    "username": "ruchamahabal"
+  },
+  {
+    "id": 104,
+    "username": "abhishekps782"
+  },
+  {
+    "id": 105,
+    "username": "rishav-sah"
+  },
+  {
+    "id": 106,
+    "username": "shubhamyadav30"
+  },
+  {
+    "id": 107,
+    "username": "ShailendraSinghRaikwar"
+  },
+  {
+    "id": 108,
+    "username": "vinaysaip"
+  }
+]
```

Comment:
GitHub API also returns the ID:

<img width="747" alt="image" src="https://user-images.githubusercontent.com/24353136/207322223-92072400-4f5b-4810-be16-142500dbf761.png">

You can skip the ID attribute and use this ID instead. Just have an array of usernames.